style.css

```diff
@@ -91,3 +91,192 @@ header {
   width: 170.17px;
   height: 20px;
 }
+
+/* works */
+#works {
+  padding-top: 114px;
+  padding-left: 24px;
+  padding-right: 24px;
+}
+
+#gridlayout {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-row-gap: 88px;
+}
+
+.worksContainer {
+  background-color: #fff;
+  box-sizing: border-box;
+  min-height: 552px;
+  padding-top: 16px;
+  padding-left: 16px;
+  padding-right: 16px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.worksImg {
+  height: 220px;
+  width: 100%;
+  border-radius: 8px;
+}
+
+.worksBodytitle {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  margin-bottom: 12px;
+}
+
+.frame2 {
+  margin-bottom: 12px;
+}
+
+.frame2 .name {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+}
+
+.dot {
+  display: inline-block;
+  width: 8px;
+  height: 8px;
+  margin-right: 12px;
+  margin-left: 12px;
+  background-color: #c1c7d0;
+  border-radius: 100%;
+}
+
+.role,
+.year {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+}
+
+.workDescription {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  margin-bottom: 12px;
+}
+
+.worksCategories {
+  display: flex;
+  list-style-type: none;
+  flex-wrap: wrap;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.worksCategories li {
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.03em;
+  color: #6070ff;
+  padding: 4px 12px;
+  margin-right: 8px;
+}
+
+.worksBtn {
+  box-sizing: border-box;
+  border: 1px solid #6070ff;
+  border-radius: 8px;
+  padding: 12px;
+  height: 48px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #396df2;
+  background-color: #fff;
+}
```

Comment:
- [ ] Good job at first 👏🏽 it could be better to add a margin-bottom to increase the distance between the button and the container to be similar to the Figma template 👇🏽 

| Figma template | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/213814734-d0a96a1d-d3d8-4028-b98f-d924711e89a8.png) ![image](https://user-images.githubusercontent.com/58642949/213814773-2341b241-02ed-49a0-a725-2f45584f1c7f.png) | ![image](https://user-images.githubusercontent.com/58642949/213814829-169ad32e-b0de-44fe-81c7-2fa3d712a6c5.png) ![image](https://user-images.githubusercontent.com/58642949/213814878-b339c1f7-7beb-4017-b59b-b3458474ae68.png) |